server/src/modules/step/use-cases/get-step.js

```diff
@@ -24,16 +24,14 @@ const getStep = async ({ id, lng, forPublic }) => {
   return {
     ...stepT,
     id: step.id,
-    checklist: [
-      stepT.thingsYouWillNeed.map((item) => ({
-        ...item,
-        stage: 'thingsYouWillNeed',
-      })),
-      stepT.whatYouWillNeedToKnow.map((item) => ({
-        ...item,
-        stage: 'whatYouWillNeedToKnow',
-      })),
-    ].flat(),
+    thingsYouWillNeed: stepT.thingsYouWillNeed.map((item) => ({
+      ...item,
+      stage: 'thingsYouWillNeed',
+    })),
+    whatYouWillNeedToKnow: stepT.whatYouWillNeedToKnow.map((item) => ({
```

Comment:
the same here : I think it's will be useful to add some (?) for stepT?. and stepT?.thingsYouWillNeed?. and also to add a backup values when it's not exit like empty array .

Reply:
done